Languages/IronPython/IronPython.Modules/socket.cs

```diff
@@ -1297,6 +1297,10 @@ public int ioctl(BigInteger cmd, object option)
             )]
         public static string getfqdn(string host) {
             host = host.Trim();
```

Comment:
I was just going by what cpython does (fails by calling .strip on None), but I guess there's no reason we can't throw a TypeError instead.